Cargo.toml

```
@@ -26,7 +26,7 @@ cfg-if = "^1.0"
 libc = "^0.2"
 
 [target.'cfg(target_os = "windows")'.dependencies]
-windows = { version = "^0.52", features = ["Win32_Foundation", "Win32_System_SystemInformation"] }
+windows = { version = "^0.54", features = ["Win32_Foundation", "Win32_System_SystemInformation"] }
```

Hey, I checked in https://github.com/svartalf/hostname/pull/28 and the code would actually be compatible with `windows` all the way up till version `0.58`.  For maximum flexibility in downstream crates, I think this should be a version range:

```suggestion
windows = { version = ">=0.52, <=0.58", features = ["Win32_Foundation", "Win32_System_SystemInformation"] }
```

(And the title updated adequately to something like ``Expand `windows` version range to include `0.58` release``).

Unfortunately it is no longer compatible since `0.59`, so I opened a separate PR for that.